README.md

```diff
@@ -1,79 +1,142 @@
+That's all well and good, but what does that mean? Well, Kubernetes clusters require some tooling to handling routing traffic from the outside world (e.g.: the Internet, corporate network, etc.) to the cluster. 
+This is typically done with a load balancer. The load balancer is responsible for routing traffic to the appropriate Kubernetes worker node which then forwards the traffic to the appropriate Service / Pod.
 
-<br/>
+If you are using a hosted web solution -- Digital Ocean, AWS, Azure, etc. -- you can use the cloud provider's load balancer service. Those services will create a load balancer for you. 
+You can use the cloud provider's API to manage the load balancer, or you can use the cloud provider's web console.
 
-## Sample Screenshots of Runtime
+However, if you checked the first box above, you are running Kubernetes on-premise and will need to manage your own load balancer. This is where NKL comes in.
 
-<br/>
+NKL itself does not perform load balancing. Instead, NKL allows you to manage resources within your cluster and have the load balancers automatically be updated to support those changes, with tooling you are most likely already using. 
 
-### Configuration with 2 Nginx LB Servers defined (HA):
+## Getting Started
 
-![NGINX LB ConfigMap](docs/media/nkl-pod-configmap.png)
+There are few bits of administrivia to get out of the way before you can start leveraging NKL for your load balancing needs.
 
-<br/>
+As noted above, NKL really shines when you have one or more Kubernetes clusters running on-premise. With this in place,
+you need to have at least one NGINX Plus host running outside your cluster (Please refer to the [Roadmap](#Roadmap) for information about other load balancer servers). 
 
-### Nginx LB Server Dashboard and Logging
+You will not need to clone this repo to use NKL. Instead, you can install NKL using the included Manifest files (just copy the `deployments/` directory), which pulls the NKL image from the Container Registry.
 
-![NGINX LB Create Nodeport](docs/media/nkl-create-nodeport.png)
+### RBAC
 
-Legend:
-- Red - kubectl commands
-- Blue - nodeport and upstreams for http traffic
-- Indigo - nodeport and upstreams for https traffic
-- Green - logs for api calls to LB Server #1
-- Orange - Nginx LB Server upstream dashboard details
-- Kubernetes Worker Nodes are 10.1.1.8 and 10.1.1.10
+As with everything Kubernetes, NKL requires RBAC permissions to function properly. The necessary resources are defined in the various YAML files in `deployement/rbac/`.
 
-<br/>
+For convenience, two scripts are included, `apply.sh`, and `unapply.sh`. These scripts will apply or remove the RBAC resources, respectively.
 
-## Requirements
+The permissions required by NKL are modest. NKL requires the ability to read Resources via shared informers; the resources are Services, Nodes, and ConfigMaps. 
+The Services and ConfigMap are restricted to a specific namespace (default: "nkl"). The Nodes resource is cluster-wide.
+
```

Comment:
I think you should include a command-by-command walkthrough here, or at least link to one in /docs

Reply:
There is a step-by-step walkthrough in the Deployment section...?

Reply:
Sounds good. I think maybe not in this PR, but in the future I might consider putting this explanation in line with the step by step since at this point I'm not sure why this is the first thing under "deployment" and I may try to run those scripts now.  However that's very subjective and I think this placement works great as well.

Reply:
I see, it's an interesting tradeoff... for me having lots of exposition between commands I'm supposed to run draws me out of focus (I may not care for all the Why if I'm just doing a rapid recon mission; but yeah, it might lead people to start prematurely.

Reply:
[Rambling, no change requested] I agree, it's very subjective.  I always want to know "why" for each command but I know a lot of people that share your take.  I like the collapse to solve this problem:

```
<details>
    <summary>Summary</summary>

    <p>Detailed content goes here …</p>
</details>
```
But I forget if it works in gh flavored markdown